charabia/src/normalizer/vietnamese.rs

```diff
@@ -1,14 +1,13 @@
 use super::{CharNormalizer, CharOrStr};
-use crate::Script;
-use crate::Token;
+use crate::{Script, Token};
 
 pub struct VietnameseNormalizer;
 
 impl CharNormalizer for VietnameseNormalizer {
     fn normalize_char(&self, c: char) -> Option<CharOrStr> {
         match c {
-            'Ð' | 'Đ' | 'đ' => Some("d".to_string().into()), // not only Vietnamese, but also many European countries use these letters
-            _ => None,
```

Comment:
None means that we remove the character, which implies that all the Latin characters were removed 😵‍💫

Reply:
Whooops!